### PR TITLE
Removed excess dependencies

### PR DESCRIPTION
**Pull request by ilya-lavrenov**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Review by xhochy — CHANGES_REQUESTED** [review]
Don't remove Python cross-compiling.

**Comment by ilya-lavrenov** [comment]
@conda-forge-admin, please rerender

**Comment by ilya-lavrenov** [comment]
@conda-forge-admin, please rerender

**Comment by ilya-lavrenov** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -26,25 +26,23 @@ requirements:
...
   run:
     - python
-    - protobuf
     - {{ pin_compatible('numpy') }}
     - typing-extensions >=3.6.2.1
 
 test:
   requires:
     - pip
+    - pytest-runner
```
Comment (xhochy) [inline recipe/meta.yaml:45]: What's the reason for this dependency?

**Pull request closed without merging**